src/auth/panel.ts

```diff
@@ -0,0 +1,45 @@
+import {LitElement, html} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+import loadingSvg from 'gmf/icons/spinner.svg';
+
+import './form.ts';
+import './auth.css';
+
+@customElement('ngeo-auth-panel')
+export default class AuthPanel extends LitElement {
+  @property({type: String}) loginInfoMessage = '';
+  @property({type: Boolean}) postLoading = false;
+
+
+  protected render() {
+    let spinnerTemplate = this.postLoading ? html`
+      <div>
+        <i class="fa fa-spin svg-lit-element">
+          ${unsafeSVG(loadingSvg)}
+        </i>
+        {{'Loading themes, please wait...' | translate}}
+      </div>
+    `: '';
+    return html`
+      <div class="row">
+        <div class="col-sm-12">
+          <div class="gmf-app-tools-content-heading">
+            {{'Login' | translate}}
+            <a class="btn close" @click=${this.closePanel}>&times;</a>
+          </div>
+          <ngeo-auth-form .login-message=${this.loginInfoMessage}></ngeo-auth-form>
+          ${spinnerTemplate}
+        </div>
+      </div>
+    `;
+  }
+  // Disable shadow DOM
+  protected createRenderRoot() {
+    return this;
+  }
+
+  closePanel() {
+    this.dispatchEvent(new CustomEvent('close-panel', {detail: false}));
+  }
+}
```

Comment:
`component.ts` isn't a better name?

Reply:
He's next to the ngeo-auth-form. Which one should be named component ? For me, it's more the "form" than the "panel". And we could have a `panel.ts` for each component in a panel in the apps.

Reply:
Yes it's my thought, I will rename form.ts and keep the panel.ts filename.